### DISCUSSION TITLE
Welcome to dezoomify Discussions!

### DISCUSSION THREAD

**Discussion by lovasoa** [lovasoa/dezoomify#536] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas, tips and tricks.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by piccachilly** [comment]
Hi lovasoa, please could you help me with the `-xargs -d ` command? I am not a coder and have been trying to figure this out myself to no avail. I've created a local `urls.txt` document with a list of URLs. 

When I try to enter `xargs -d '\n' -n 1 ./dezoomify-rs </Users/my-computer/urls.txt` it fails. I know I'm doing something clearly incorrectly. 

Any help gratefully received.
